projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-tags/documenten-api-tags.component.html

```diff
@@ -0,0 +1,93 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<valtimo-carbon-list
+  *ngIf="{
+    loading: loading$ | async,
+    documentTags: documentTags$ | async,
+    addModalClosed: addModalClosed$ | async
+  } as obs"
+  [items]="obs.documentTags"
+  [fields]="fields$ | async"
+  [header]="false"
+  [loading]="obs.loading"
+  [actionItems]="ACTION_ITEMS"
+  [showSelectionColumn]="true"
+  [isSearchable]="true"
+  [pagination]="pagination"
+  (paginationClicked)="paginationClicked($event)"
+  (paginationSet)="paginationSet($event)"
+  (search)="searchTermEntered($event)"
+>
+  <ng-container carbonToolbarContent *ngTemplateOutlet="buttons"></ng-container>
```

Comment:
Why not put the add button template directly here?

Reply:
Because it's used twice